src/include/mallocMC/distributionPolicies/XMallocSIMD.hpp

```diff
@@ -64,8 +73,127 @@ namespace mallocMC
          *        default configuration. The default can be obtained through
          *        XMallocSIMD<>::Properties
          */
-        template<typename T_Config = XMallocSIMDConf::DefaultXMallocConfig>
```

Comment:
please do not remove `XMallocSIMDConf::DefaultXMallocConfig`

Reply:
Well spotted, thx!

Reply:
Well spotted, thx!